services/QuillLMS/spec/services/units/creator_spec.rb

```diff
@@ -5,11 +5,11 @@
 Sidekiq::Testing.fake!
 
 describe Units::Creator do
```

Comment:
This file was a small cleanup that is unrelated (I attempted to trigger the deadlock from many different angles). Leaving it in, since it's slightly cleaner.